js/transform.js

```diff
@@ -222,6 +232,15 @@ function displayForm(e) {
 }
 
 function generateAndDisplayForm(formId, dest) {
+    const INTEGER_AFTER_UNDERSCORE_REGEX = /(.*)_([0-9]+)/;
+    const match = formId.match(INTEGER_AFTER_UNDERSCORE_REGEX);
+    if (match) {
+        var formName = match[1]
+        var formIterator = match[2]
```

Comment:
[Dust] The name `formIterator` feels a bit strange to me, mainly because I think of iterators as objects like arrays. Wondering if `formKey` or `formId` or `formIndex` would be better? Or maybe there's a specific reason here

Reply:
I like `formIndex`!